startup.py

```diff
@@ -21,6 +21,6 @@
     elif choice == '2':
         dbmanager.stop()
     elif choice == '3':
-        os.system("python3 initdb.py")
+        initdb.main("","")
```

Comment:
If I am not mistaken you can just do `initdb.main()` because I have default parameters in the main function.